include/SDL_hints.h

```diff
@@ -2048,6 +2048,20 @@ extern "C" {
  */
 #define SDL_HINT_WINDOWS_DPI_SCALING "SDL_WINDOWS_DPI_SCALING"
 
+/**
+ * \brief Opts created windows into the Windows DWM dark mode appearance.
+ *
+ *  By default, Windows does not opt any applications into adhering to the dark
+ *  mode color scheme for title bars/borders. This hint tells Windows to use
+ *  the appropriate dark/light mode colors for created SDL windows.
+ *
+ *  This variable can be set to the following values:
+ *    "0"       - Windows will always use the light mode color scheme.
+ *    "1"       - Windows will follow the user light/dark mode preference and use
+ *                dark title bar and window borders when appropriate.
+ */
+#define SDL_HINT_WINDOWS_DWM_DARK_MODE "SDL_WINDOWS_DWM_DARK_MODE"
```

Comment:
I wonder if we should make a cross-platform hint for this instead of making it Windows-only.

Reply:
Maybe, though I don't know what the implementation would look like for any other platforms.